azure-test/tests/azure_search_service/test-get-expected.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "id": "{{ output.resource_id.value }}",
+    "name": "{{ resourceName }}",
+    "provisioning_state": "succeeded",
+    "region": "east us",
```

Comment:
we should minimize hard coded value in expected json. we can set output values and then use.

Reply:
also remove empty value checks.